resources/lang/en/common/lookup/departments.php

```diff
@@ -10,14 +10,54 @@
     *
     */
 
-    "treasury_board" => "Treasury Board of Canada Secretariat",
-    "natural_resources" => "Natural Resources Canada",
-    "transport" => "Transport Canada",
-    "enviroment_and_climate_change" => "Environment and Climate Change Canada",
-    "employment_and_social_development" => "Employment and Social Development Canada",
-    "global_affairs" => "Global Affairs Canada",
-    "fisheries_and_oceans" => "Fisheries and Oceans Canada",
-    "innovation_science" => "Innovation, Science and Economic Development Canada",
-    "public_service_and_procurement" => "Public Services and Procurement Canada",
-    "border_services_agency" => "Canada Border Services Agency",
+    'departments' => [
```

Comment:
If the only place these values are used is the AddTranslatedNameAndImpactToDepartments migration, its probably better to define these values in that migration file.

Reply:
Removed.